Userland/Shell/Builtin.cpp

```diff
@@ -39,6 +39,43 @@ int Shell::builtin_dump(int argc, char const** argv)
     return 0;
 }
 
+int Shell::builtin_where(int argc, char const** argv)
+{
+    bool at_least_one_succeded = false;
+    for (ssize_t i = 1; i < argc; i++) {
```

Comment:
Let's use Core::ArgsParser like the rest of the builtins.

Reply:
Thanks for the feedback. Done.